ui/src/models/rollout/generated/api.ts

```diff
@@ -524,6 +524,12 @@ export interface GithubComArgoprojArgoRolloutsPkgApisRolloutsV1alpha1CanaryStep
      * @memberof GithubComArgoprojArgoRolloutsPkgApisRolloutsV1alpha1CanaryStep
      */
     setCanaryScale?: GithubComArgoprojArgoRolloutsPkgApisRolloutsV1alpha1SetCanaryScale;
```

Comment:
i dont think this file require an update. is this a merge issue?

Reply:
Although I didnot change anything on these it is coming up as a conflict when PR is raised so I had add these to fix the conflict.